From 01ff69a6d9e1f2fd70d929422fa2a2e56f044c0e Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Sun, 8 Jan 2017 11:19:08 +0100
Subject: [PATCH] library/tvshows: switch 'genre' and 'recently added'

This makes 'recently added' the first entry.

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>

library/movies: switch 'genre' and 'recently added'

This makes 'recently added' the first entry.

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>

estuary: move 'recently added' entries to the top in home screen

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 addons/skin.estuary/xml/Home.xml                   | 24 +++++++++++-----------
 system/library/video/movies/genres.xml             |  2 +-
 .../library/video/movies/recentlyaddedmovies.xml   |  2 +-
 system/library/video/tvshows/genres.xml            |  2 +-
 .../video/tvshows/recentlyaddedepisodes.xml        |  2 +-
 5 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/addons/skin.estuary/xml/Home.xml b/addons/skin.estuary/xml/Home.xml
index 0d04a02..3faec7e 100644
--- a/addons/skin.estuary/xml/Home.xml
+++ b/addons/skin.estuary/xml/Home.xml
@@ -57,18 +57,18 @@
 							<param name="list_id" value="5900"/>
 						</include>
 						<include content="WidgetListPoster" condition="Library.HasContent(movies)">
-							<param name="content_path" value="special://skin/playlists/inprogress_movies.xsp"/>
-							<param name="widget_header" value="$LOCALIZE[31010]"/>
-							<param name="widget_target" value="videos"/>
-							<param name="list_id" value="5100"/>
-						</include>
-						<include content="WidgetListPoster" condition="Library.HasContent(movies)">
 							<param name="content_path" value="special://skin/playlists/recent_unwatched_movies.xsp"/>
 							<param name="widget_header" value="$LOCALIZE[20386]"/>
 							<param name="widget_target" value="videos"/>
 							<param name="list_id" value="5200"/>
 						</include>
 						<include content="WidgetListPoster" condition="Library.HasContent(movies)">
+							<param name="content_path" value="special://skin/playlists/inprogress_movies.xsp"/>
+							<param name="widget_header" value="$LOCALIZE[31010]"/>
+							<param name="widget_target" value="videos"/>
+							<param name="list_id" value="5100"/>
+						</include>
+						<include content="WidgetListPoster" condition="Library.HasContent(movies)">
 							<param name="content_path" value="special://skin/playlists/unwatched_movies.xsp"/>
 							<param name="widget_header" value="$LOCALIZE[31007]"/>
 							<param name="widget_target" value="videos"/>
@@ -117,6 +117,12 @@
 							<param name="widget_target" value="videos"/>
 							<param name="list_id" value="6900"/>
 						</include>
+						<include content="WidgetListEpisodes" condition="Library.HasContent(tvshows)">
+							<param name="content_path" value="special://skin/playlists/recent_unwatched_episodes.xsp"/>
+							<param name="widget_header" value="$LOCALIZE[20387]"/>
+							<param name="widget_target" value="videos"/>
+							<param name="list_id" value="6200"/>
+						</include>
 						<include content="WidgetListPoster" condition="Library.HasContent(tvshows)">
 							<param name="content_path" value="videodb://inprogresstvshows"/>
 							<param name="sortby" value="lastplayed"/>
@@ -125,12 +131,6 @@
 							<param name="widget_target" value="videos"/>
 							<param name="list_id" value="6100"/>
 						</include>
-						<include content="WidgetListEpisodes" condition="Library.HasContent(tvshows)">
-							<param name="content_path" value="special://skin/playlists/recent_unwatched_episodes.xsp"/>
-							<param name="widget_header" value="$LOCALIZE[20387]"/>
-							<param name="widget_target" value="videos"/>
-							<param name="list_id" value="6200"/>
-						</include>
 						<include content="WidgetListPoster" condition="Library.HasContent(tvshows)">
 							<param name="content_path" value="special://skin/playlists/unwatched_tvshows.xsp"/>
 							<param name="widget_header" value="$LOCALIZE[31122]"/>
diff --git a/system/library/video/movies/genres.xml b/system/library/video/movies/genres.xml
index f3fdf26..94fa1d3 100644
--- a/system/library/video/movies/genres.xml
+++ b/system/library/video/movies/genres.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
-<node order="1" type="filter">
+<node order="10" type="filter">
   <label>135</label>
   <icon>DefaultGenre.png</icon>
   <content>movies</content>
diff --git a/system/library/video/movies/recentlyaddedmovies.xml b/system/library/video/movies/recentlyaddedmovies.xml
index 1363d4b..943dc8f 100644
--- a/system/library/video/movies/recentlyaddedmovies.xml
+++ b/system/library/video/movies/recentlyaddedmovies.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
-<node order="10" type="folder" visible="Library.HasContent(Movies)">
+<node order="1" type="folder" visible="Library.HasContent(Movies)">
   <label>20386</label>
   <icon>DefaultRecentlyAddedMovies.png</icon>
   <path>videodb://recentlyaddedmovies/</path>
diff --git a/system/library/video/tvshows/genres.xml b/system/library/video/tvshows/genres.xml
index 69b4dcd..e4e586a 100644
--- a/system/library/video/tvshows/genres.xml
+++ b/system/library/video/tvshows/genres.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
-<node order="1" type="filter">
+<node order="11" type="filter">
   <label>135</label>
   <icon>DefaultGenre.png</icon>
   <content>tvshows</content>
diff --git a/system/library/video/tvshows/recentlyaddedepisodes.xml b/system/library/video/tvshows/recentlyaddedepisodes.xml
index caf1357..e0e4dac 100644
--- a/system/library/video/tvshows/recentlyaddedepisodes.xml
+++ b/system/library/video/tvshows/recentlyaddedepisodes.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
-<node order="11" type="folder" visible="Library.HasContent(TVShows)">
+<node order="1" type="folder" visible="Library.HasContent(TVShows)">
   <label>20387</label>
   <icon>DefaultRecentlyAddedEpisodes.png</icon>
   <path>videodb://recentlyaddedepisodes/</path>
-- 
2.0.1

